prisma/schema.prisma

```diff
@@ -34,13 +35,28 @@ model Password {
 }
 
 model Group {
-  id          String  @id @default(uuid())
+  id          String             @id @default(uuid())
   name        String
-  description String  @db.Text
+  description String             @db.Text
   imgUrl      String?
   imgAlt      String?
   events      Event[]
   users       User[]
+  socalLinks  GroupSocialLink[]
+}
+
+model GroupSocialLink {
+  groupId String @id @unique
+  group   Group  @relation(fields: [groupId], references: [id], onDelete: Cascade)
+  type    String
```

Comment:
Maybe this could be an enum? Are we actually planning to enforce types or should we just remove this?